src/components/DuosHeader.jsx

```diff
@@ -0,0 +1,540 @@
+                      to={{ pathname: tab.link, state: { selectedMenuTab: tabIndex } }}
+                      component={Link}
+                    />
+                  ))}
+                </Tabs>
+              </Box>
+            )
+          }
+          {
+            !isLogged && (
+              <ul className="navbar-public">
+                <li>
+                  <a
+                    id="link_about"
+                    className="navbar-duos-link"
+                    href="https://broad-duos.zendesk.com/hc/en-us/articles/360060400311-About-DUOS"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    <div className="navbar-duos-icon-about" style={navbarDuosIcon}></div>
+                    <span style={navbarDuosText}>About</span>
+                  </a>
+                </li>
+                <li>
+                  <a
+                    id="link_help"
+                    className="navbar-duos-link"
+                    href="https://broad-duos.zendesk.com/hc/en-us/articles/360059957092-Frequently-Asked-Questions-FAQs-"
+                    target="_blank" rel="noreferrer"
+                  >
+                    <div className="navbar-duos-icon-help" style={navbarDuosIcon}></div>
+                    <span style={navbarDuosText}>FAQs</span>
+                  </a>
+                </li>
+                {contactUsButton}
+                {supportrequestModal}
+              </ul>
+            )
+          };
+        </div>
+        {isLogged && (
+          <div
+            style={{ display: 'flex', alignItems: 'center', flexDirection: orientation === 'vertical' ? 'column' : 'row' }}
+          >
+            <button onClick={showRequestModal} style={styles.navButton}>
+              <div id="help" style={{ whiteSpace: 'nowrap' }}>Contact Us</div>
+            </button>
+            {supportrequestModal}
+            <li className="dropdown user-li" style={{ listStyleType: 'none' }}>
+              <a id="sel_user" role="button" className="dropdown-toggle" data-toggle="dropdown">
+                <div id="dacUser">
+                  {currentUser.displayName}
+                  <span className="caret caret-margin"></span>
+                </div>
+                <small id="dacUserMail">{currentUser.email}</small>
+              </a>
+              <ul className="dropdown-menu navbar-dropdown" role="menu" style={{ top: orientation === 'vertical' ? '-100%' : '100%' }}>
+                <li>
+                  <Link id="link_profile" to="/profile" onClick={onSubtabChange}>Your Profile</Link>
+                </li>
+                <li>
+                  <a id="link_signOut" onClick={signOut}>Sign out</a>
+                </li>
+              </ul>
+            </li>
+          </div>
+        )}
+      </ul>
+
+      {tabs[selectedMenuTab]?.children && (
+        <Box className="duos-navigation-box navbar-sub">
```

Comment:
nit: add back comments
```suggestion
        // Sub Tabs
        <Box className="duos-navigation-box navbar-sub">
```